met-web/src/services/widgetService/DocumentService.tsx/index.tsx

```diff
@@ -15,13 +15,15 @@ export const fetchDocuments = async (widget_id: number): Promise<DocumentItem[]>
 };
 
 interface PostDocumentRequest {
-    name: string;
-    widget_id: number;
+    name?: string;
```

Comment:
i don't think any of these should be nullable, they are all required for creating a document

Reply:
Agreed, react hook form however defines them as string | undefined. Even when you specify they are required
![image](https://user-images.githubusercontent.com/91914654/203654605-634a6032-06b3-44ad-90ba-3c19d1b7beec.png)


not entirely sure how to improve that